loleaflet/src/layer/tile/CanvasTileLayer.js

```diff
@@ -107,7 +107,7 @@ L.CanvasTilePainter = L.Class.extend({
 	},
 
 	canvasDPIScale: function () {
-		return parseInt(this._canvas.width) / this._width;
+		return parseInt(this._canvas.width) / parseFloat(this._canvas.style.width.replace('px', ''));
```

Comment:
This is not the first time I see the "parseFloat(this._canvas.style.width.replace('px', ''))" - shouldn't we have an own method for this (and the height)?